cmd/nex/cli.go

```diff
@@ -11,10 +11,12 @@ type Globals struct {
 	GlobalLogger `prefix:"logger." group:"Logger Configuration"`
 	GlobalNats   `prefix:"nats." group:"NATS Configuration"`
 
-	Config    kong.ConfigFlag  `help:"Configuration file to load" placeholder:"./nex.config.json"`
-	Version   kong.VersionFlag `help:"Print version information"`
-	Namespace string           `env:"NEX_NAMESPACE" placeholder:"default" help:"Specifies namespace when running nex commands"`
-	Check     bool             `help:"Print the current configuration"`
+	Config              kong.ConfigFlag  `help:"Configuration file to load" placeholder:"./nex.config.json"`
+	Version             kong.VersionFlag `help:"Print version information"`
+	Namespace           string           `env:"NEX_NAMESPACE" placeholder:"default" help:"Specifies namespace when running nex commands"`
```

Comment:
maybe this should get moved out of globals and only into the commands where it applies? It's potentially confusing since it doesn't apply to the `node up` command